__tests__/integration/profilesLoad.test.js

```diff
@@ -19,8 +19,11 @@ describe('Profiles load', () => {
 
     // await page.screenshot({path: 'screenshot.png', fullPage: true})
 
-    const ptitle_sel = 'div#bfeditor-formdiv > form > div > h4'
-    const profile_title = await page.$eval(ptitle_sel, e => e.textContent)
-    await expect(profile_title).toMatch(/BIBFRAME Instance/)
+    await expect_regex_in_selector_textContent('div#bfeditor-formdiv > form > div > h4', /^BIBFRAME Instance/)
```

Comment:
Note that this one does need to be a regex because "Clone Instance" text also turns up inside the h4.  :-P